packages/vscode-extension/package.nls.json

```diff
@@ -157,6 +157,30 @@
     "teamstoolkit.commandsTreeViewProvider.manageCollaboratorTitle": "Manage Collaborator",
     "teamstoolkit.commandsTreeViewProvider.manageCollaboratorDescription": "Manage M365 Teams App (with Microsoft Entra app) Collaborators",
     "teamstoolkit.commandsTreeViewProvider.addWebpartTitle": "Add SPFx Web Part",
+    "teamstoolkit.commandsTreeViewProvider.officeDevDeployTitle": "Deploy",
+    "teamstoolkit.commandsTreeViewProvider.officeDevDeployDescription": "Deploy",
+    "teamstoolkit.commandsTreeViewProvider.publishAppSourceTitle": "Publish",
+    "teamstoolkit.commandsTreeViewProvider.publishAppSourceDescription": "Link to the wiki about how to publish the add-in to AppSource",
+    "teamstoolkit.commandsTreeViewProvider.createOfficeAddInTitle": "Create a New Add-in",
+    "teamstoolkit.commandsTreeViewProvider.createOfficeAddInDescription": "Create a new add-in project of Word Excel or Powerpoint",
+    "teamstoolkit.commandsTreeViewProvider.checkAndInstallDependenciesTitle": "Check and Install Dependencies",
+    "teamstoolkit.commandsTreeViewProvider.checkAndInstallDependenciesDescription": "Check and install dependencies",
+    "teamstoolkit.commandsTreeViewProvider.officeDevLocalDebugTitle": "Preview Your Add-in (F5)",
+    "teamstoolkit.commandsTreeViewProvider.officeDevLocalDebugDescription": "Local debug your Add-in App",
+    "teamstoolkit.commandsTreeViewProvider.validateManifestTitle": "Validate Manifest",
+    "teamstoolkit.commandsTreeViewProvider.validateManifestDescription": "Validate the manifest of Office add-ins project",
+    "teamstoolkit.commandsTreeViewProvider.editManifestTitle": "Edit Manifest",
+    "teamstoolkit.commandsTreeViewProvider.editManifestDescription": "Edit the manifest of Office add-ins project",
+    "teamstoolkit.commandsTreeViewProvider.generateManifestGUIDTitle": "Generate Manifest GUID",
+    "teamstoolkit.commandsTreeViewProvider.generateManifestGUIDDescription": "Generate new manifest GUID",
+    "teamstoolkit.commandsTreeViewProvider.officeAddIn.officePartnerCenterTitle": "Open Partner Center",
+    "teamstoolkit.commandsTreeViewProvider.officeAddIn.officePartnerCenterDescription": "Open Partner Center",
+    "teamstoolkit.commandsTreeViewProvider.officeAddIn.getStartedTitle": "Get Started",
+    "teamstoolkit.commandsTreeViewProvider.officeAddIn.getStartedDescription": "Get Started about how to create Office Add-in project",
+    "teamstoolkit.commandsTreeViewProvider.officeAddIn.documentationTitle": "Documentation",
+    "teamstoolkit.commandsTreeViewProvider.officeAddIn.documentationDescription": "The documentation about how to create Office Add-in project",
+    "teamstoolkit.commandsTreeViewProvider.officeAddIn.stopDebugTitle": "Stop Preview Your Office Add-in",
```

Comment:
Proposed: Stop Preview of Your Office Add-in

Reply:
fixed.